src/main/java/io/jenkins/plugins/analysis/warnings/IarCstat.java

```diff
@@ -0,0 +1,54 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.parser.IarCstatParser;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+
+/**
+ * Provides a parser and customized messages for the IAR C-Stat static analysis tool.
+ *
+ * @author Lorenz Aebi
+ */
+public class IarCstat extends ReportScanningTool {
+    private static final long serialVersionUID = 6672928932731913714L;
+    static final String ID = "iarCstat";
```

Comment:
Nomuppercase in URLs. Use ˋiar-cstatˋ or something like that.

Reply:
corrected